index.bs

```diff
@@ -202,26 +202,15 @@ The OrientationSensor Interface {#orientationsensor-interface}
 
 <pre class="idl">
   interface OrientationSensor : Sensor {
-    void populateQuaternion(Float32Array targetBuffer);
+    readonly attribute FrozenArray<<double>>? quaternion;
```

Comment:
@kenchris what example do you have in mind? Quats are not that useful on their own, good example would be getting rotation matrix or Euler angles. Or you mean example like in Issue #21 ?